valheim_plus.cfg

```diff
@@ -232,6 +238,22 @@ dataRate=60
 ; The interval in seconds that the game auto saves at (client only)
 autoSaveInterval=1200
 
+[Sleep]
+enabled=false
+; Change false to true to enable this section
+
+byPlayers= false
+; Change to true to sleep when a number of players are sleeping
+
+numberOfPlayersToSleep = 1
```

Comment:
Use percentage always, can introduce unnecessary issues, and if they don't know how a simple percent works, they should not be hosting servers. Period.